### ISSUE TITLE
genetest.tests returns errors on Mac OSX

### ISSUE THREAD

**Issue by mvacher** [pgxcentre/genetest#16]
Hi,

After installing genetest following the instruction from the documentation, I've tried to run the tests but got the following errors:

I'd love to use this script for my analyses but can't figure out how to fix this issue.

Thanks
```
python -m genetest.tests
/usr/local/anaconda3/lib/python3.7/site-packages/yaml/constructor.py:126: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  if not isinstance(key, collections.Hashable):
.........snp4: Singular matrix
.snp4: Singular matrix
snp3: Singular matrix
snp2: Singular matrix
snp1: Singular matrix
x./usr/local/anaconda3/lib/python3.7/site-packages/statsmodels/duration/hazard_regression.py:873: RuntimeWarning: divide by zero encountered in true_divide
  hess += xp2 * np.sum(1 / c0)
/usr/local/anaconda3/lib/python3.7/site-packages/statsmodels/duration/hazard_regression.py:873: RuntimeWarning: invalid value encountered in multiply
  hess += xp2 * np.sum(1 / c0)
/usr/local/anaconda3/lib/python3.7/site-packages/statsmodels/duration/hazard_regression.py:874: RuntimeWarning: invalid value encountered in true_divide
  hess -= xp2f * np.sum(J / c0)
/usr/local/anaconda3/lib/python3.7/site-packages/statsmodels/duration/hazard_regression.py:875: RuntimeWarning: invalid value encountered in true_divide
  mat = (xp1[None, :] - np.outer(J, xp1f)) / c0[:, None]
/usr/local/anaconda3/lib/python3.7/site-packages/statsmodels/duration/hazard_regression.py:758: RuntimeWarning: invalid value encountered in true_divide
  ratio = numer / denom
/usr/local/anaconda3/lib/python3.7/site-packages/statsmodels/base/optimizer.py:271: RuntimeWarning: invalid value encountered in greater
  oldparams) > tol)):
/usr/local/anaconda3/lib/python3.7/site-packages/numpy/core/_methods.py:28: RuntimeWarning: invalid value encountered in reduce
  return umr_maximum(a, axis, None, out, keepdims, initial)
/usr/local/anaconda3/lib/python3.7/site-packages/numpy/core/_methods.py:28: RuntimeWarning: invalid value encountered in reduce
  return umr_maximum(a, axis, None, out, keepdims, initial)
/usr/local/anaconda3/lib/python3.7/site-packages/scipy/stats/_distn_infrastructure.py:879: RuntimeWarning: invalid value encountered in greater
  return (self.a < x) & (x < self.b)
/usr/local/anaconda3/lib/python3.7/site-packages/scipy/stats/_distn_infrastructure.py:879: RuntimeWarning: invalid value encountered in less
  return (self.a < x) & (x < self.b)
/usr/local/anaconda3/lib/python3.7/site-packages/scipy/stats/_distn_infrastructure.py:1821: RuntimeWarning: invalid value encountered in less_equal
  cond2 = cond0 & (x <= self.a)
x........./usr/local/anaconda3/lib/python3.7/site-packages/genetest/statistics/descriptive.py:43: RuntimeWarning: Mean of empty slice
  maf = np.nanmean(genotypes) / 2
..............EEEEEsnp5: condition number is large, inf
.snp5: condition number is large, inf
........../usr/local/anaconda3/lib/python3.7/site-packages/statsmodels/regression/linear_model.py:1633: RuntimeWarning: divide by zero encountered in double_scalars
  return np.sqrt(eigvals[0]/eigvals[-1])
..snp3: Perfect separation detected, results not available
.snp3: Perfect separation detected, results not available
....../usr/local/anaconda3/lib/python3.7/site-packages/statsmodels/genmod/families/family.py:880: RuntimeWarning: invalid value encountered in true_divide
  n_endog_mu = self._clean((1. - endog) / (1. - mu))
................................................sssss
======================================================================
ERROR: test_categorical_gwas (genetest.tests.test_gwaswriter.TestGWASWriter)
Tests a categorical GWAS analysis.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/tests/test_gwaswriter.py", line 146, in test_categorical_gwas
    output_prefix="",
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/analysis.py", line 219, in execute_formula
    modelspec, subgroups = modelspec_from_formula(formula, test, test_kwargs)
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/modelspec/__init__.py", line 69, in modelspec_from_formula
    modelspec = ModelSpec(**model)
TypeError: __init__() got an unexpected keyword argument 'conditions'

======================================================================
ERROR: test_complex_interaction_categorical_gwas (genetest.tests.test_gwaswriter.TestGWASWriter)
Tests a complex interaction GWAS analysis.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/tests/test_gwaswriter.py", line 279, in test_complex_interaction_categorical_gwas
    output_prefix=self.output_name,
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/analysis.py", line 219, in execute_formula
    modelspec, subgroups = modelspec_from_formula(formula, test, test_kwargs)
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/modelspec/__init__.py", line 69, in modelspec_from_formula
    modelspec = ModelSpec(**model)
TypeError: __init__() got an unexpected keyword argument 'conditions'

======================================================================
ERROR: test_complex_interaction_gwas (genetest.tests.test_gwaswriter.TestGWASWriter)
Tests a complex interaction GWAS analysis.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/tests/test_gwaswriter.py", line 226, in test_complex_interaction_gwas
    output_prefix=self.output_name,
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/analysis.py", line 219, in execute_formula
    modelspec, subgroups = modelspec_from_formula(formula, test, test_kwargs)
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/modelspec/__init__.py", line 69, in modelspec_from_formula
    modelspec = ModelSpec(**model)
TypeError: __init__() got an unexpected keyword argument 'conditions'

======================================================================
ERROR: test_normal_gwas (genetest.tests.test_gwaswriter.TestGWASWriter)
Tests a normal GWAS analysis.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/tests/test_gwaswriter.py", line 111, in test_normal_gwas
    output_prefix="",
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/analysis.py", line 219, in execute_formula
    modelspec, subgroups = modelspec_from_formula(formula, test, test_kwargs)
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/modelspec/__init__.py", line 69, in modelspec_from_formula
    modelspec = ModelSpec(**model)
TypeError: __init__() got an unexpected keyword argument 'conditions'

======================================================================
ERROR: test_simple_interaction_gwas (genetest.tests.test_gwaswriter.TestGWASWriter)
Tests a simple interaction GWAS analysis.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/tests/test_gwaswriter.py", line 189, in test_simple_interaction_gwas
    output_prefix="",
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/analysis.py", line 219, in execute_formula
    modelspec, subgroups = modelspec_from_formula(formula, test, test_kwargs)
  File "/usr/local/anaconda3/lib/python3.7/site-packages/genetest/modelspec/__init__.py", line 69, in modelspec_from_formula
    modelspec = ModelSpec(**model)
TypeError: __init__() got an unexpected keyword argument 'conditions'

----------------------------------------------------------------------
Ran 114 tests in 6.752s

FAILED (errors=5, skipped=5, expected failures=2)
```

**Comment by lemieuxl** [comment]
This is a known issue (caused by newer version of `grako`). We'll release a fix soon.

**Comment by lemieuxl** [comment]
This should be fixed in release 0.5.0.
